pkgs/tools/security/diceware/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "diceware";
+  version = "0.9.3";
+  name = "${pname}-${version}";
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.

Reply:
Basically, the latest release has a bug in a test: the test [checks for the current year's date](https://github.com/ulif/diceware/blob/cbdfbac8e6c7d8c7c6dff027e4d8ab2714b1dfa2/tests/test_diceware.py#L258), but the code [hasn't been updated yet](https://github.com/ulif/diceware/commit/5f37b234aea5252863f604ed1227608e1d5536b4) (the update is still unreleased).

So I can either fetch a more recent version, patch the code, patch the test, or disable tests.

Which way is best?

Reply:
I think you should definitely patch

Reply:
However, that fix isn't really a fix, as I think it will break again in 2019.

Reply:
Patch the code or the test?

Reply:
Well the best solution would be to open an upstream PR that solves this problem so that it also won't break in the years to come.
The second best solution is to simply remove/disable the specific test, open an upstream issue about the problem with future breakages and a comment that links to that issue.

Reply:
The problem is solved, but on master, not in the released version. So is it better to just build a version that has the fix?

Reply:
Ah I see, so an updated version wouldn't break in 2019?
Then I'd say just apply that specific patch (or patches) until a new version is released.